CMSgov/mint-app
# [EASI-4378] Add Missing custom solution name

# EASI-4378


## Changes and Description

- removed conditional render of the custom solution details form as `otherHeader` in custom solution names is not in use
- code clean up




## How to test this change

1. Create a custom solution
2. Update the custom solution
3. Ensure when name of custom solution is there before you edit it
4. Verify data persist after save
5.  🎉 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
